Bottom: eccac8ed1a003568f8e9abae3d174d8d538360d6
Top:    c209d2408434b68cc97462775ae12fbbbf10da0c
Author: Robert Collins <robertc@robertcollins.net>
Date:   2013-03-23 01:49:06 +1300

Refresh of threadsafestreamresult

---

diff --git a/testtools/tests/test_testresult.py b/testtools/tests/test_testresult.py
index ea1d5af..9726c4b 100644
--- a/testtools/tests/test_testresult.py
+++ b/testtools/tests/test_testresult.py
@@ -1693,7 +1693,7 @@ class TestMergeTags(TestCase):
             expected, _merge_tags(current_tags, changing_tags))
 
 
-class TestThreadStreamResult(TestCase):
+class TestStreamToQueue(TestCase):
 
     def make_result(self):
         queue = Queue()
